js/core/utils.js

```diff
@@ -14,6 +14,61 @@ define(
                 msg.pub("end", "core/utils");
                 cb();
             }
+        ,   normalizePadding: function (text) {
+                function testRegex(regex) {
+                    return function(text) {
+                        return regex.test(text);
+                    }
+                }
+                if (!text) {
+                    return "";
+                }
+                if (typeof text !== "string") {
+                    throw TypeError("Invalid input");
+                }
+                var hasOpeningPre = testRegex(/<pre\b[^>]*>/i);
```

Comment:
I'll give this some thought... I could work with a live DOM, and check if `.parentElement` is `<pre>`